package/boot/uboot-mvebu/patches/0004-arm-mvebu-clearfog-upgrade-sata-speed-to-sata-3-6Gbp.patch

```diff
@@ -0,0 +1,47 @@
+From 75b24bae6bcb7e33ee9b564d751e0523602ad040 Mon Sep 17 00:00:00 2001
+From: Josua Mayer <josua@solid-run.com>
+Date: Sat, 23 Dec 2023 21:30:24 +0100
+Subject: [PATCH 4/5] arm: mvebu: clearfog: upgrade sata speed to sata-3
+ (6Gbps)
+
+According to Marvell documentation Amrada 385 and 388 support up to
+sata-3. Upgrade the serdes rates accordingly.
+
+Signed-off-by: Josua Mayer <josua@solid-run.com>
+---
+ board/solidrun/clearfog/clearfog.c | 6 +++---
+ 1 file changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/board/solidrun/clearfog/clearfog.c b/board/solidrun/clearfog/clearfog.c
+index 6fa2fe5fe3..2ee98a3020 100644
+--- a/board/solidrun/clearfog/clearfog.c
++++ b/board/solidrun/clearfog/clearfog.c
+@@ -51,7 +51,7 @@ static void cf_read_tlv_data(void)
+ 
+ /* The starting board_serdes_map reflects original Clearfog Pro usage */
+ static struct serdes_map board_serdes_map[] = {
+-	{SATA0, SERDES_SPEED_3_GBPS, SERDES_DEFAULT_MODE, 0, 0},
++	{SATA0, SERDES_SPEED_6_GBPS, SERDES_DEFAULT_MODE, 0, 0},
```

Comment:
Was this maybe done intentionally? Please check the commit history of u-boot.
Maybe the chip has problems with higher speeds or some boards have problems with signal integrity. 

Does this only affect u-boot r also the Linux starting next?

Reply:
Lrom what I saw this boot-time setting affects Linux, there appears to be no dynamic reconfiguration of the serdes speed on armada-38x in Linux.
Linux will still be able to negotiate sata-2 or sata-1 though.

U-Boot history reveals nothing, it may simply have been overlooked during initial bringup.